### PR TITLE
fix pip install by applying boundaries to implicit dependencies

### PR DESCRIPTION
**Pull request by suzhoum**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/setup.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "evaluate>=0.4.0,<0.5.0",
     "timm>=0.9.5,<0.10.0",
     "torchvision>=0.16.0,<0.19.0",
+    "tifffile>=2024.1.30",
```
Comment (Innixma) [inline multimodal/setup.py:40]: Add a in-line comment for why this is needed

File: timeseries/setup.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "accelerate",  # version range defined in `core/_setup_utils.py`
     "gluonts==0.15.1",
     "networkx",  # version range defined in `core/_setup_utils.py`
+    "statsmodels>=0.14.2,<0.15",
```
Comment (Innixma) [inline timeseries/setup.py:35]: Add a in-line comment for why this is needed

File: timeseries/setup.py [hunk 3]
```diff
@@ -58,7 +59,9 @@
         "optimum[openvino,nncf]>=1.17,<1.19",
     ],
     "chronos-onnx": [  # for faster CPU inference in pretrained models with ONNX
-        "optimum[onnxruntime]>=1.17,<1.19",
+        "optuna>=3.4.0,<5.0.0",
+        "onnx>=1.15.0",
+        "optimum[onnxruntime]>=1.20,<1.22.1",
```
Comment (Innixma) [inline timeseries/setup.py:64]: Add a in-line comment for why this is needed

**Comment by suzhoum** [comment]
Close due to root cause identified in: https://github.com/explosion/spaCy/issues/13653

**Pull request closed without merging**